pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -402,6 +416,8 @@ func TestReconciler(t *testing.T) {
 		wantJob           batchv1.Job
 		wantWorkloads     []kueue.Workload
 		wantErr           error
+		checkWlOwner      bool
+		skipOwnerUpdates  bool
```

Comment:
I think we don't need it, just skip the adoption in test when `PrebuiltWorkloadLabel` label is set.